### PR TITLE
ENCD-5492-5491-5497-Clean up some config and deployment

### PR DESCRIPTION
**Pull request by caseylitton**

### CODE REVIEW

File: cloud-config/run-scripts/ami-preapp.sh [hunk 1]
```diff
@@ -10,6 +10,10 @@ fi
 echo -e "\n\t$ENCD_INSTALL_TAG $(basename $0) Running"
 
 # Script Below
+# Update ulimit hard and soft openfile/nofile limits
+echo '* soft nofile 4096' | sudo -u root tee -a /etc/security/limits.conf
+echo '* hard nofile 4096' | sudo -u root tee -a /etc/security/limits.conf
```
Comment (caseylitton) [inline cloud-config/run-scripts/ami-preapp.sh]: unrelated testing, need to remove

File: dev-config.ini [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[deployment]
+conf_dir = ./cloud-config
```
Comment (caseylitton) [inline dev-config.ini]: For temporary backwards compatibility the default config repo is set to  the repo's cloud-config.  To override, and use the devops versions,  copy this file to .dev-config.ini and change conf_dir = location-of-devops-cloud-config.

Eventually, the dev-config.ini will default to a demonstration deployment and not our internal aws based cloud deployment.

**Review by caseylitton — COMMENTED** [review]
notes

File: cloud-config/run-scripts/ami-preapp.sh (hunk 1 above)
Comment (gabdank) [reply cloud-config/run-scripts/ami-preapp.sh]: I am confused, if this is unrelated testing, why is it here?

**Review by gabdank — CHANGES_REQUESTED** [review]
I think the lines in ami-preapp.sh need to be removed, and tests are failing.

File: cloud-config/run-scripts/ami-preapp.sh (hunk 1 above)
Comment (caseylitton) [reply cloud-config/run-scripts/ami-preapp.sh]: I did a self review of my PR and found some code I forgot to remove.  I flagged it so other devs will know to ignore it.

**Review by keenangraham — DISMISSED** [review]

**Review by gabdank — DISMISSED** [review]

**Review by keenangraham — DISMISSED** [review]
ENCD-5491 approved.

File: README.md [hunk 3]
```diff
@@ -50,6 +50,23 @@ For issues see [Snovault OSX App Installation][] first.
     $ ./circle-tests.sh npm
     ```
 
+## Configuration Installation (In Progress)
+    The configuration repo is set by 'conf_dir = ./cloud-config' in the 'dev-config.ini' file.  The
+    'deploy.py' compiles yaml files from 'cloud-config' for use with aws instances.  There is a
+    very specific file structure at the moment.  We are working to generalize the use case.
+
+    To use a different directory for cloud configuration copy 'dev-config.ini' to '.dev-config.ini'.
```
Comment (gabdank) [inline README.md]: I think here it should be demo-config.ini and not dev-config.ini

**Review by gabdank — CHANGES_REQUESTED** [review]
you need to update the README.md

File: README.md (hunk 3 above)
Comment (gabdank) [reply README.md]: @caseylitton here is the comment in the PR I left you yesterday

**Review by gabdank — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (caseylitton) [reply README.md]: updated.

**Review by gabdank — DISMISSED** [review]

**Review by gabdank — DISMISSED** [review]

**Review by gabdank — APPROVED** [review]

**Pull request merged**